tests/conftest.py

```diff
@@ -18,6 +19,13 @@
 CACHE_ARTIFACTS_SCRIPT_PATH = root + "/tests/cache_artifacts.sh"
 
 
+def pytest_sessionfinish():
```

Comment:
Compile tries to log a bunch of stuff using the `atexit` decorator.  However, pytest closes these logs before they finish so it throws an I/O error. 

This disables logging exceptions. Not sure if the right way to do it.